README.md

```diff
@@ -1,4 +1,88 @@
-# test2
+# school-library
 
-test
-.
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [school-library] <a name="about-project"></a>
+
+A library system that provides serveices for students and teachers
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Use oop**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Getting Started
+
+\*\*Please follow the steps below to setup this project in your local computer
+
+### Prerequisites
+
+A basic understanding of Ruby and Git
+
+### Usage
+
+1. Install Ruby
+2. Clone this repo `https://github.com/mohisa302/school-library.git`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors
+
+👤 **Mohi**
+
+- GitHub: [@mohisa302](https://github.com/mohisa302)
+- Twitter: [@Kholochelam](https://twitter.com/Kholochelam)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mohadese-sadeghi-692551199/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **add more methods**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The link to MIT is broken. It will be great to add an MIT license to your project. This will make your project even more professional.

Reply:
This change has been done well